src/license_expression/__init__.py

```diff
@@ -599,6 +599,35 @@ def simple_tokenizer(self, expression):
                         sym = LicenseSymbol(key=sym_or_op)
                     yield Token(start, end, sym_or_op, sym)
 
+    def dedup(self, expression):
+        """
+        Return a de-dupulicated expression
+        """
+        l = Licensing()
+        exp = l.parse(expression)
+        operator = vars(exp)['operator']
+        dedup_expression = ''
+        expression_list = []
```

Comment:
May be we could use just `expressions` as a name ?

Reply:
@pombredanne do you mean change the function name from`dedup` to `expressions` ?

Reply:
Updated the function name from `dedup` to `expressions`
https://github.com/nexB/license-expression/pull/53/commits/43da938cc4822e184c067795bf5e0fee468a6193